controllers/memberController.js

```diff
@@ -139,7 +142,8 @@ exports.getMembers = catchAsync(async (req, res, next) => {
     mlWing,
     designWing,
     literaryWing,
-    coHeads,
+    prWing,
+    Heads,
```

Comment:
Let it remain `coHeads`, else changes must be done on the frontend logic as well